False positive for D417

Running ruff v0.0.124, python 3.10.6, on OSX 12.6.1

Linting the following code:

```python
def f(x):
    """Do f.

    Args:
        x: the value

    Returns: the value
    """
    return x
```

Reports 
```
test.py:1:1: D417 Missing argument description in the docstring: `x`
```

The error goes away if the `Returns` block is reformatted with a new line.

```python
def f(x):
    """Do f.

    Args:
        x: the value

    Returns:
        the value
    """
    return x
```

--- a/src/pydocstyle/plugins.rs
+++ b/src/pydocstyle/plugins.rs
@@ -1359,7 +1359,10 @@ static GOOGLE_ARGS_REGEX: Lazy<Regex> =
 
 fn args_section(checker: &mut Checker, definition: &Definition, context: &SectionContext) {
     let mut args_sections: Vec<String> = vec![];
-    for line in textwrap::dedent(&context.following_lines.join("\n")).lines() {
+    for line in textwrap::dedent(&context.following_lines.join("\n"))
+        .trim()
+        .lines()
+    {
         if line
             .chars()
             .next()

